resources/lang/en/auth.php

```diff
@@ -15,5 +15,20 @@
 
     'failed' => 'These credentials do not match our records.',
     'throttle' => 'Too many login attempts. Please try again in :seconds seconds.',
+    'remember-me' => 'Remember Me',
+    'login' => 'Login',
+    'Logout' => 'logout',
```

Comment:
```suggestion
    'Logout' => 'Logout',
```

Reply:
c'est l'inverse non ?

```suggestion
    'logout' => 'Logout',
```

Reply:
Indeed ! J'approuve